README.md

```diff
@@ -142,6 +142,19 @@ possible to dynamically modify the logging at runtime.
 - `http://$RAINBOW_CTL_LISTEN_ADDRESS/mgr/log/level?subsystem=<system name or * for all system>&level=<level>` will set the logging level for a subsystem
 - `http://$RAINBOW_CTL_LISTEN_ADDRESS/mgr/log/ls` will return a comma separated list of available logging subsystems
 
+## Purging Peer Connections
+
+Connections to a specific peer, or to all peers, can be closed and the peer information removed from the peer store. This can be useful to help determine if the presence/absence of a connection to a peer is affecting behavior. Be aware that purging a connection is inherently racey as it is possible for the peer to reestablish a connection at any time following a purge.
+
```

Comment:
@gammazero we should note how this feature interacts with `RAINBOW_DHT_SHARED_HOST`   / `cfg.DHTSharedHost`. I think this won't show DHT peers, and we only list bitswap peers, so maybe:


```suggestion

If `RAINBOW_DHT_SHARED_HOST=false` this endpoint will not show peers connected to DHT host, and only list ones used for Bitswap.

```